gradle.properties

```diff
@@ -1 +1 @@
-version=1.3.22-SNAPSHOT
+version=2.0.00-SNAPSHOT
```

Comment:
We are bumping the version to 2.0.0

Which Reactor BOM are we targeting?

cc\ @OlegDokuka

Reply:
I wonder more why we need to bump to 2.0?

Reply:
This probably doesn't need to be 2.0, 1.4 would be ok, I think (as I mentioned in the issue).

It's probably too late to get it into 2023.0.0 since the RC is tomorrow so, I guess, the next one.

Reply:
Or, we could just keep it as 1.3.x (just that we don't normally do that in Spring projects, but this is not Spring).

Reply:
The current reactor-kafka 1.3.x is included in Reactor 2020.0.x, 2022.0.x and 2023.0.x (to be released in a month).
We expect backwards compatibility.
If you keep the backwards compatibility then it should be OK

Reply:
Also having a new version means branching ...

Reply:
Right; since there are some minor API changes, they might break backwards compat.; we may be able to work around it; I'll have to take a closer look, but I doubt I will have time before tomorrow's release train.

Reply:
@garyrussell lets chat and see whether we need 1.4

Reply:
@OlegDokuka After further review; all these changes are in test classes; there are no breaking API changes or backwards compatibility issues; so I think we can stick with the 1.3.x line.

Reply:
I have reverted the changes back to the 1.3.x version line.